src/Components/TodoItem.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/TodoItem.module.css';
+
+const TodoItem = ({
+  itemProp, setTodos, delTodo, setUpdate,
+}) => {
+  const [editing, setEditing] = useState(false);
+
+  const handleChange = (id) => {
+    setTodos((prevState) => prevState.map((todo) => {
+      if (todo.id === id) {
+        return {
+          ...todo,
+          completed: !todo.completed,
+        };
+      }
+      return todo;
+    }));
+  };
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
+
+  const viewMode = {};
+  const editMode = {};
+  if (editing) {
+    viewMode.display = 'none';
+  } else {
+    editMode.display = 'none';
+  }
+
+  return (
+    <li className={styles.item}>
+      <div
+        className={styles.content}
+        style={viewMode}
+      >
+        <input
+          type="checkbox"
+          checked={itemProp.completed}
+          onChange={() => {
+            handleChange(itemProp.id);
+          }}
+        />
+        <button
+          type="button"
+          onClick={handleEditing}
+        >
+          Edit
+        </button>
```

Comment:
Please consider fixing the position of this Edit button. 

![Screen Shot 2023-03-08 at 16 03 43](https://user-images.githubusercontent.com/44959066/223724418-5f027e50-30f4-4406-b8fe-58e70c657a46.png)

Reply:
@mertkantaroglu   The button work perfectly